### ISSUE TITLE
Investigate minification of js/css/html during deployment

### ISSUE THREAD

**Issue by jakemac53** [dart-lang/core-elements#181]
Mangling names would be hard due to the js-interop, but we should be able to save a lot of space by just removing comments and collapsing whitespace.

**Comment by jakemac53** [comment]
Whatever solution we use here should also be applied to https://github.com/dart-lang/paper-elements

**Comment by Andersmholmgren** [comment]
Be nice if there is any way to apply tree shaking to the elements. Remove unused elements etc

**Comment by jakemac53** [comment]
I agree, but we can't really do that until we have global transformers (see http://dartbug.com/17306). The application is the only thing that knows what it could delete, but it can't delete files in this package.

**Comment by zoechi** [comment]
Shouldn't it be possible to just run vulcanize or some other JavaScript minification after `pub build` to achive this?

**Comment by jakemac53** [comment]
Yes, but it would be nice if we could integrate that into the transformation process, eliminating any extra steps and standardizing how its done.
